<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>604</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Datenbankanwendung</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="actionBEenden"/>
   </widget>
   <widget class="QMenu" name="menuStammdaten">
    <property name="title">
     <string>&amp;Stammdaten</string>
    </property>
    <addaction name="actionPostleitzahlen"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>&amp;Extras</string>
    </property>
    <addaction name="actionPLZImportiern"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuStammdaten"/>
   <addaction name="menuExtras"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <action name="actionBEenden">
   <property name="text">
    <string>B&amp;eenden</string>
   </property>
  </action>
  <action name="actionPostleitzahlen">
   <property name="text">
    <string>&amp;Postleitzahlen</string>
   </property>
  </action>
  <action name="actionPLZImportiern">
   <property name="text">
    <string>PLZ &amp;Importiern...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
